pkg/channel/message_receiver_test.go

```diff
@@ -218,7 +220,12 @@ func TestMessageReceiver_ServerStart_trace_propagation(t *testing.T) {
 	server := httptest.NewServer(kncloudevents.CreateHandler(r))
 	defer server.Close()
 
-	require.NoError(t, tracing.SetupStaticPublishing(logger.Sugar(), "localhost", tracing.AlwaysSample))
+	require.NoError(t, tracing.SetupStaticPublishing(logger.Sugar(), "localhost", &tracingconfig.Config{
+		Backend:        tracingconfig.Zipkin,
+		Debug:          true,
+		SampleRate:     1.0,
+		ZipkinEndpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans",
```

Comment:
I assume this is a copy&paste from somewhere, and this is only test code, but I'd personally prefer if we didn't hard code the istio-system here in eventing.

Reply:
This is really just to provide a valid placeholder: the point is just to let the code pass through the tracing code paths